cmake/external_rules.cmake

```diff
@@ -90,6 +92,153 @@ function(download_external_sources)
     OUTPUT_FILE ${PROJECT_BINARY_DIR}/external/output_cmake_build.txt
     WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/external
   )
+
+  if(NOT ANDROID AND NOT IOS)
```

Comment:
Is this equivalent to easier to read`if(DESKTOP)`?

Reply:
I don't know if DESKTOP is necessarily set here since it's in a function, but I'll use the external_platform value set above.

Reply:
Oh whoops, that's in a different function. Anyway I'll put a comment in, but because we aren't guaranteed to have DESKTOP set (since it's set in a different cmake file) I'd rather just check this here, as it's safer.